android/app/src/main/res/layout/activity_confirm_empty.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do you actually use this layout anywhere? If not, please remove from pull request.